src/zarr/api/synchronous.py

```diff
@@ -523,6 +525,29 @@ def open_group(
     )
 
 
+def create_group(
+    store: StoreLike,
+    *,
+    path: str | None = None,
+    zarr_format: ZarrFormat | None = None,
+    overwrite: bool = False,
+    attributes: dict[str, Any] | None = None,
+    storage_options: dict[str, Any] | None = None,
+) -> Group:
```

Comment:
we should paste in a docstring based on the async version once we are happy with that one